### PR TITLE
rename IsValidating to IsValid

### PR DESCRIPTION
**Pull request by JaredCorduan**

### CODE REVIEW

File: alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs [hunk 1]
```diff
@@ -161,16 +165,23 @@ import qualified Shelley.Spec.Ledger.UTxO as Shelley
 
 -- ===================================================
 
+{-# DEPRECATED IsValidating "Use IsValid instead" #-}
+type IsValidating = IsValid
+
+{-# DEPRECATED isValidating "Use isValid instead" #-}
+isValidating :: ValidatedTx era -> IsValid
+isValidating = isValid
```
Comment (nc6) [inline alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs:173]: For completeness we should also have a deprecated `HasField` instance.

**Review by nc6 — APPROVED** [review]
One minor comment

**Review by nc6 — APPROVED** [review]
One minor comment

File: alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs (hunk 1 above)
Comment (JaredCorduan) [reply alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs:173]: great catch, thank you!

Comment (JaredCorduan) [reply alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs:173]: @nc6 is it possible to deprecate a class instance? It doesn't seem like it to me. I can add the instance, and leave a TODO to remove it when we remove the other deprecated functions.

**Pull request merged**